controllers/movieController.js

```diff
@@ -0,0 +1,47 @@
+const Movie 		= require("./../models/Movie.model")
+
+exports.getAllMovies = async (req, res) => {
+	
+	const allMovies = await Movie.find({})
+
+	console.log(allMovies)
+
+	res.render("movies/movies", {
+		data: allMovies
+	})
+}
+
+exports.getMovie = async (req, res) => {
+
+	const singleMovieID = req.params.movieID
+
+	const getTheMovie = await Movie.findById(singleMovieID)
+
+	console.log(getTheMovie)
+
+	res.render("movies/singleMovie", {
+		data: getTheMovie
+	})
+}
+
+exports.viewCreateMovie = async (req, res) => {
+
+	res.render("movies/create")
+
+}
+
+exports.createMovie = async (req,res) => {
+const title = req.body.title
```

Comment:
Utilizando destructuracion podemos hacer lo siguente y eliminar las lineas siguentes 
```suggestion
  const { title, director, image, stars, description, showtimes } = req.body;
```